vars/util.groovy

```diff
@@ -198,11 +198,18 @@ GIT INFORMATION REPORT
 
 def getNextSnapshot(String version)
 {
```

Comment:
Please use same formatting as the rest of the file, which means opening brackets at the end of the previous line